conversiontracerbullet/src/main/java/org/stanford/MarcToXML.java

```diff
@@ -37,12 +37,12 @@
  */
 class MarcToXML {
 
-    private static Connection authDB = null;
+    public static Connection authDB = null;
```

Comment:
Yes, they are public so the tests can see them. I tried using reflection, but couldn't get it to work very easily. I will try 'package' or 'protected' and see if either of those work.